docs/changelog/1_0_a5.rst

```diff
@@ -0,0 +1,60 @@
+===========
+1.0 Alpha 5
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 5 "Luhman 16".
```

Comment:
I'd remove the "16" as it looks confusing with all the previous numbers.

Reply:
+1

Reply:
Done